src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/AppxModuleHelper.cs

```diff
@@ -419,6 +452,109 @@ private async Task InstallVCLibsDependenciesAsync()
             }
         }
 
+        private async Task<bool> TryInstallDependenciesFromGitHubArchive(string releaseTag)
+        {
+            try
+            {
+                var githubClient = new GitHubClient(RepositoryOwner.Microsoft, RepositoryName.WinGetCli);
+                var release = await githubClient.GetReleaseAsync(releaseTag);
+
+                var dependenciesJsonAsset = release.GetAsset(DependenciesJsonName);
+                var dependenciesZipAsset = release.GetAsset(DependenciesZipName);

```

Comment:
When this gets replaced with a `TryGetAsset`, we should still allow successful "install" of dependencies if only the json is available and all requirements are met.